### PR TITLE
Initial cdk

### PR DESCRIPTION
**Pull request by andrew-nowak**
## What does this change?

Set up cdk package for this project, bring along the feeds stack previously defined in the old repo

### CODE REVIEW

File: cdk/lib/wires-feeds.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { Queue } from 'aws-cdk-lib/aws-sqs';
+
+export type WiresFeedsProps = GuStackProps;
+const app = 'wires-feeds';
+
+export class WiresFeeds extends GuStack {
+	constructor(scope: App, id: string, props: WiresFeedsProps) {
+		super(scope, id, { ...props, app });
+
+		const stackStageApp = `${this.stage}/${this.stack}/${app}`;
```
Comment (bryophyta) [inline cdk/lib/wires-feeds.ts:17]:
```suggestion
		const stageStackApp = `${this.stage}/${this.stack}/${app}`;
```

File: cdk/lib/wires-feeds.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+		const fingerpostPublishingUserArn = new GuStringParameter(
+			this,
+			'fingerpost-publishing-user-arn',
+			{
+				fromSSM: true,
+				default: `/${stackStageApp}/fingerpost-publishing-user-arn`,
```
Comment (bryophyta) [inline cdk/lib/wires-feeds.ts:24]:
```suggestion
				default: `/${stageStackApp}/fingerpost-publishing-user-arn`,
```

**Review by bryophyta — APPROVED** [review]
lgtm! just had a suggestion about variable naming.

the one other issue I spotted is that my VSCode isn't happy with the `tsconfig`, though I'm assuming that's just a local problem so I'm happy to play around later and see if I can work out what's going on.

**Review by m75gyqbsrw-boop — COMMENTED** [review]
noop

**Pull request merged**